code/style.css

```diff
@@ -1,5 +1,279 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+@media (max-width: 668px) {
```

Comment:
We usually start with the default styling and then put the media queries further down.